src/silx/gui/plot/items/shape.py

```diff
@@ -133,7 +135,7 @@ def getType(self):
         """
         return self._type
 
-    def getPoints(self, copy=True):
+    def getPoints(self, copy=True) -> numpy.ndarray:
         """Get the control points of the shape.
 
         :param bool copy: True (Default) to get a copy,
```

Comment:
```suggestion
    def getPoints(self, copy: bool = True) -> numpy.ndarray:
        """Get the control points of the shape.

        :param copy: True (Default) to get a copy,
```

+ Remove `:rtype` below